cylc/flow/suite_files.py

```diff
@@ -1278,7 +1277,9 @@ def check_flow_file(
         if flow_file_path.is_symlink():
             # Symlink broken or points elsewhere - replace
             flow_file_path.unlink()
-        flow_file_path.symlink_to(suite_rc_path)
+        flow_file_path.symlink_to(
+            suite_rc_path.relative_to(
+                flow_file_path.parent))
```

Comment:
Why not
```suggestion
        flow_file_path.symlink_to(SuiteFiles.SUITE_RC)
```

Reply:
Thanks for these suggestions @MetRonnie, I can't commit them as had rebased but did manually add them.